plugins/org.eclipse.elk.alg.rectpacking/src/org/eclipse/elk/alg/rectpacking/p2packing/NoPlacement.java

```diff
@@ -0,0 +1,72 @@
+/*******************************************************************************
+ * Copyright (c) 2022 sdo and others.
+ * 
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ * 
+ * SPDX-License-Identifier: EPL-2.0 
+ *******************************************************************************/
+package org.eclipse.elk.alg.rectpacking.p2packing;
+
+import java.util.List;
+
+import org.eclipse.elk.alg.rectpacking.RectPackingLayoutPhases;
+import org.eclipse.elk.alg.rectpacking.options.InternalProperties;
+import org.eclipse.elk.alg.rectpacking.options.RectPackingOptions;
+import org.eclipse.elk.alg.rectpacking.util.DrawingUtil;
+import org.eclipse.elk.core.alg.ILayoutPhase;
+import org.eclipse.elk.core.alg.LayoutProcessorConfiguration;
+import org.eclipse.elk.core.math.ElkPadding;
+import org.eclipse.elk.core.math.KVector;
+import org.eclipse.elk.core.util.IElkProgressMonitor;
+import org.eclipse.elk.graph.ElkNode;
+
+/**
+ * Strategy that just does nothing with the input and just calculates the drawing size based on the coordinates of the
+ * children. Whitespace elimination cannot be done with this packing strategy since it might not succeed.
```

Comment:
Since it is impossible to eliminate whitespace for general packings and I do not want to add a whitespace eliminator just yet that can deal with general graphs.